tests/e2e/fixtures/ncc/tsconfig.json

```diff
@@ -1,5 +1,7 @@
 {
   "compilerOptions": {
+    "target": "ES2020",
+    "module": "commonjs",
```

Comment:
Why is this being changed?

Reply:
To fix this issue 

<img width="1149" alt="image" src="https://user-images.githubusercontent.com/4535142/164530073-69e76a60-8695-4fcc-8de9-42e7b025189a.png">

Reply:
This should be a separate PR

Reply:
And how I will pass the test?

Reply:
I'm not sure. If there is no other way to import dindest here and make it work then I guess your solution is ok. But a comment would be useful. If it is just for dindest then for example the comment would be "TODO: remove this once we stop using dindest or fix it".

I am also surprised that this used to work and now does not. What changed? 🤔

Reply:
I removed the `yarn.lock`😢 to fix the first issue then we built new versions. 
maybe updating the packages did that 
![image](https://user-images.githubusercontent.com/4535142/164536443-db56ab64-899c-4c2e-8f00-83f7412095a2.png)